scenarios/COLD_START.md

```diff
@@ -0,0 +1 @@
+# Managing Cold Start Scenarios in Recommendation Systems

```

Comment:
Do we know what to add in these markdown files?

Reply:
The idea is to explain scenarios and match them with the algos that we have in the repo

Reply:
Yea there is a concrete example about what we want to add (maybe we can start with a particular topic, e.g., news), and then the rest can be added based on it.